ebooks/mastering-mongoose-javascript-jabber.html

```diff
@@ -278,7 +278,7 @@ <h3>What's Inside</h3>
 
 
     <script type="text/javascript">
-      var stripeKey = 'pk_test_51IkuAqIFSwo5WpGWudAKEeemrymI6EmICEAgkgvlq4Bo5jJ1uuMRlrBRw9kvHH7boANqjE7Y6Mb7lQmsXRQoZo3x00Ek1L6d8A';
+      var stripeKey = 'pk_live_51IoDkvEIfgvH77vtfJZwToZlC20FilHNRViMce42TBCeFsptEIkUcjz09MMPbfn4pU9St2nM8P9aYlL4l7q8SLbf00ivnWmIpw';
```

Comment:
I'm wondering why this change keeps happening... We should really add these to gitignore